server/src/main/java/org/opensearch/cluster/coordination/CoordinationState.java

```diff
@@ -651,7 +651,7 @@ public interface PersistedState extends Closeable {
          * Returns the stats for the persistence layer for {@link CoordinationState}.
          * @return PersistedStateStats
          */
-        PersistedStateStats getStats();
+        PersistedStateStats getUploadStats();
```

Comment:
I think we should keep the base method name as `getStats` only. This method is overridden in RemotePersistedState, LucenePersistedState, InMemoryPersistedState. Other than RemotePersistedState, upload stats does not make sense in other classes.

Reply:
makes sense, updated it.